### ISSUE TITLE
CT101 thermostat not updating values until 'Refresh' or re-interview is clicked in the UI manually

### ISSUE THREAD

**Issue by automaton82** [AlCalzone/node-zwave-js#6938]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Snap

### Z-Wave JS UI version

9.12.0

### ZwaveJS version

12.9.1

### Describe the bug

For some reason with the latest version of z-wave JS UI, when my thermostat updates the operating state (such as when the AC or furnace come on), there is no update given to z-wave JS UI until either 'Refresh' is clicked, or a re-interview is done manually in the UI. The moment this is done, all values are updated properly.

### To Reproduce

Use the CT101 thermostat, join it in a network and then after it's running, change the temperature until it kicks on the furnace or AC. The value for 'Operating State v2' will not update even if you wait indefinitely, nor will the temperature.

### Expected behavior

This should update like before.

### Additional context

This has happened to me before with earlier versions but it always sorted itself out after re-interviewing. With the latest version nothing I do seems to fix it.

Here's an example log from today:

```
6/14/2024, 1:57:09 PM - VALUE UPDATED
Arg 0:
└─commandClassName: Thermostat Operating State
└─commandClass: 66
└─property: state
└─endpoint: 0
└─newValue: 2
└─prevValue: 0
└─propertyName: state
6/14/2024, 1:37:39 PM - INTERVIEW COMPLETED
6/14/2024, 1:37:39 PM - READY
6/14/2024, 1:37:39 PM - INTERVIEW STAGE COMPLETED
Arg 0:
Complete
6/14/2024, 1:37:39 PM - INTERVIEW STAGE COMPLETED
Arg 0:
OverwriteConfig
```

Notice I did a manual re-interview at 1:37pm. At this point, `Operating State` was `Idle`. I then kicked on the AC and let it run for about 10 minutes, but z-wave JS UI did not register the change.

I then manually clicked `Refresh` in the UI which caused the update at 1:57pm and showed the correct value, `Cooling`.

**Comment by robertsLando** [comment]
@automaton82 This looks to me like a device issue, does it used to work with previous versions so?

Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

The log should contain both a re-interview and you doing something that should result in a report.

**Comment by automaton82** [comment]
I think you're right, I think it's a CT101 device issue. I think it doesn't push out value updates and must be polled.

**Comment by Indianb0y016** [comment]
I can chime in as I have a Ct100 and used to use a CT101.
I can confirm that I recently began having issues where the device would send out its temperature values on every change. The fan state would send pretty regularly, with a few times being missed here and there. 
However, as of late, none of the parameters are receiving updates at all, even with various troubleshooting steps applied. In fact, I am actually running into some other issues with other nodes not having full node data points, like my zooz q sensors.
I am investigating a little further and am gonna create a separate issue if needed, but I can assure that the CT100 SHOULD send out temperature updates regularly, including fan state (usually); assuming the associations are configured correctly.

Edit: Turns out the job of identifying the breakers in my house yesterday messed up the nodes, because the network infrastructure and all respective machines were powered down. I troubleshooted the issues I have been dealing with by hard resetting problematic nodes. The Q sensors are working fine now, and the CT100 is reporting temperature changes correctly now, after pressing the reset button.

**Comment by apella12** [comment]
fwiw - I have the Ct101 and can verify the issue.  It does always report when the room temperature changes, but other items like Op state, fan, etc are hit and miss.  I have a poll rule setup based on the room temperature and the hysteresis. For instance if with 1 degree swing and heat if the temp drops 1 degree below the setpoint and the Op state is not Heat after 4 seconds, I poll it.  Also basically poll when I change the setpoint, if it doesn't turn on.  I keep track of the polls to make sure I'm not flooding the network. It varies by season, but somewhere between 10 and 40 a day are required.

**Comment by PeteRager** [comment]
Maybe unrelated but I am seeing problems with polling in the latest release, in that after a period of time, polls just get dropped.  I'm debugging to see which version the problem starts occurring in.

**Comment by Hilo4321** [comment]
Thought I'd toss a log in here since I was fixing something else, and I see this behavior also. 

I'd call out that this does update sometimes, but generally I don't rely on it because it is so sporadic.

I changed the setpoint, waited for the heat to kick on, saw the status didn't update, then refreshed.

Log also contains an interview post update. Ignore log prior to 18:30 (log time) or so. I updated from v13.0 around then.

[zwavejs_current.log](https://github.com/user-attachments/files/17548622/zwavejs_current.log)

**Comment by Hilo4321** [comment]
@robertsLando sorry for the ping, but I'm not sure how to reopen this. Seems like I don't have permission.

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by mikedm139** [comment]
I'm running into this same issue with a CT100 thermostat. Here's a debug log with a re-interview and adjusting the thermostat setpoint to get the furnace to kick in.
For reference in the log, the thermostat is Node 007
[zwavejs_2026-01-29.log](https://github.com/user-attachments/files/24950547/zwavejs_2026-01-29.log)

If there's more I can do to help debug or troubleshoot, I'll do my best.

**Comment by AlCalzone** [comment]
I don't see anything wrong in that log. Associations are set up correctly, the interview completes, setting and querying the setpoint works fine.

**Comment by mikedm139** [comment]
@AlCalzone Thanks for having a look. The thermostat seems to work fine. I can control it via my Home Assistant instance integrated via Zwave-js-ui. However, it doesn't automatically update the Operating State (CC 66). I know it used to do so although I don't know at what point it stopped doing so. It only occurred to me recently since I was going to create an automation that involved tracking the HVAC time in an active state.

**Comment by apella12** [comment]
IMO this is a device issue, not anything zwave-js can solve (see my comment from awhile back). Possibly a MFG decision to not flood the network with everything that could change when a furnace/AC turns on. Typically only 2 variables are changed at a time. The temperature is reliable with every change (at least for me after several years) and that can be used in scripts/rules (with the hysteresis) to refresh op state or whatever else you are interested in. I have no interest in the zw-js project, except as a user, so I'm not just saying this for ulterior reasons.

**Comment by mikedm139** [comment]
@apella12 That makes sense. It just seems odd that the behaviour used to be there and now is not. I haven't changed the thermostat and I don't think there have been any firmware updates between when it was previously working and whenever it stopped.

**Comment by apella12** [comment]
At least for me it has always acted this way, but maybe I wasn't always paying that close of attention (or cared). Once I also started to use zniffer I noticed it.  Also have not always used zwave-js, but had the same observations with another ZW program, so that is why I think it is the device. Device has other quirks with humidity reporting and inconsistent use of EPs. If this device actually fails will not buy another RTC product, but for now this is the devil I know ;-)

**Comment by automaton82** [comment]
Yes, I have also always had this (thus the bug entry). I agree at this point I think it's the device, not zwavejs.

To 'fix' it, I have an automation running every few minutes with:

```
data:
  entity_id:
    - climate.z_wave_thermostat_3
  refresh_all_values: true
action: zwave_js.refresh_value
```

It's been working fine for years.

**Comment by mikedm139** [comment]
@automaton82 Thanks for the suggested automation. I guess I'll implement something similar. If the consensus is that the behaviour is a device shortcoming then I will quit trying to solve it and just work around it. I still thinks it's strange that my unit used to update operating state reliably and now it doesn't but I can live with the workaround.
